yaml/landuse.yaml

```diff
@@ -1,114 +1,482 @@
-  - filter: 
-      landuse: forest
-      protect_class: '6'
-    min_zoom: GREATEST(LEAST(zoom, 14), 7)
-    output: {kind: forest}
-  - filter: {landuse: forest}
-    min_zoom: GREATEST(LEAST(zoom, 14), 9)
-    output: {kind: forest}
-  - filter: {landuse: residential}
+    output: {kind: carousel}
+  - filter: {tags->attraction: amusement_ride}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: residential}
-  - filter: {landuse: retail}
+    output: {kind: amusement_ride}
+  - filter: {historic: fort}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: retail}
+    output: {kind: fort}
+
+  ############################################################
+  # TIER 1
+  ############################################################
+  - filter:
+      historic: battlefield
+      not: { operator: *us_forest_service }
+    min_zoom: *tier1_min_zoom
+    output:
+      kind: battlefield
+      tier: 1
+  - filter:
+      boundary: national_park
+      not: { operator: *us_forest_service }
+    min_zoom: *tier1_min_zoom
+    output:
+      kind: national_park
+      tier: 1
+
+  ############################################################
+  # TIER 2
+  ############################################################
+  - filter:
+      boundary: national_park
+      not: { operator: *us_parks_service }
+    min_zoom: *tier2_min_zoom
+    output:
+      kind: national_park
+      tier: 2
+  - filter:
+      any:
+        leisure: park
+        landuse: park
+      not: { operator: *us_parks_service }
+    min_zoom: *tier2_min_zoom
+    output:
+      kind: park
+      tier: 2
+
+  - filter: { landuse: forest, operator: *us_forest_service }
+    min_zoom: { max: [ 6, { lit: *tier2_min_zoom } ] }
+    output: { kind: forest, tier: 2 }
+  - filter: { landuse: forest }
+    min_zoom: { max: [ 10, { lit: *tier2_min_zoom } ] }
+    output: { kind: forest, tier: 2 }
+
+  - filter: {boundary: protected_area}
+    min_zoom: *tier2_min_zoom
+    output: {kind: protected_area, tier: 2}
+  - filter: {leisure: nature_reserve}
+    min_zoom: *tier2_min_zoom
+    output: {kind: nature_reserve, tier: 2}
+
+  - filter: { landuse: wood, operator: *us_forest_service }
+    min_zoom: { max: [ 6, { lit: *tier2_min_zoom } ] }
+    output: { kind: wood, tier: 2 }
+  - filter: { landuse: wood }
+    min_zoom: { max: [ 10, { lit: *tier2_min_zoom } ] }
+    output: { kind: wood, tier: 2 }
+
+  - filter: { natural: forest, operator: *us_forest_service }
+    min_zoom: { max: [ 6, { lit: *tier2_min_zoom } ] }
+    output: { kind: natural_forest, tier: 2 }
+  - filter: { natural: forest }
+    min_zoom: { max: [ 10, { lit: *tier2_min_zoom } ] }
+    output: { kind: natural_forest, tier: 2 }
+
+  - filter: { natural: wood, operator: *us_forest_service }
+    min_zoom: { max: [ 6, { lit: *tier2_min_zoom } ] }
+    output: { kind: natural_wood, tier: 2 }
+  - filter: { natural: wood }
+    min_zoom: { max: [ 10, { lit: *tier2_min_zoom } ] }
```

Comment:
Just checking: you want natural woods without USFS operator to show up only at zoom 14?